<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1071</width>
      <height>401</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>94</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>30</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>36</number>
    </attribute>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Device</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Speed</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Voltage(p1)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Current(p1)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P (p1)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q (p1)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Voltage(p2)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Current(p2)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P (p2)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q (p2)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Voltage(p3)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Current(p3)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P (p3)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q (p3)</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>Ali</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1087</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
